### PR TITLE
Ui testing

### PR DESCRIPTION
**Pull request by appleshill**

### CODE REVIEW

File: src/test/java/seedu/address/ui/CommandBoxTest.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    }
+
+    /**
+     * A mock of the CommandExecutor interface for testing purposes.
+     */
+    @FunctionalInterface
+    private interface CommandExecutor {
+        CommandResult execute(String commandText) throws CommandException, ParseException;
+    }
+}
```
Comment (valhrd) [inline src/test/java/seedu/address/ui/CommandBoxTest.java:141]: I would never have thought of this, awesome implementation!

File: src/test/java/seedu/address/ui/PersonCardTest.java [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+            return text;
+        }
+
+        public void setText(String newText) {
+            this.text = newText;
+        }
+
+
+    }
+}
```
Comment (valhrd) [inline src/test/java/seedu/address/ui/PersonCardTest.java:125]: LGTM and very complete!

**Review by valhrd — COMMENTED** [review]
LGTM and good tests for the UI.

File: src/test/java/seedu/address/ui/CommandBoxTest.java [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
```
Comment (DiegoTheExplorar) [inline src/test/java/seedu/address/ui/CommandBoxTest.java:11]: Thank you very much for taking on the task of adding UI testing. It is a very laborious process!

**Review by DiegoTheExplorar — APPROVED** [review]

**Pull request merged**